wpilib_interface/build.gradle

```diff
@@ -5,58 +5,117 @@ import com.vanniktech.maven.publish.JavadocJar
 
 plugins {
 	id "java"
+	id "maven-publish"
```

Comment:
we should not have this in the build.gradle now that we have `id "com.vanniktech.maven.publish" version "0.30.0"`